go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/celestiaorg/celestia-openrpc
 
-go 1.20
+go 1.21.5
```

Comment:
```suggestion
go 1.21
```

Reply:
➤ go mod tidy -go=1.21                                                                             14:31:39
go: github.com/celestiaorg/go-square@v1.0.1 requires go@1.21.5, but 1.21 is requested